src/media_tools/webvtt.c

```diff
@@ -430,7 +443,7 @@ GF_Err wvtt_dump(GF_Box *a, FILE * trace)
 	GF_WebVTTSampleEntryBox *cuebox = (GF_WebVTTSampleEntryBox *)a;
 	gf_isom_box_dump_start(a, "WebVTTSampleEntryBox", trace);
 	fprintf(trace, ">\n");
-	if (cuebox->config) boxstring_dump((GF_Box *)cuebox->config, trace);
+	//if (cuebox->config) boxstring_dump((GF_Box *)cuebox->config, trace);
```

Comment:
You're right, absolutely.
I forgot about that while tampering to figure out where the duplication came from. Removed it in dbe7aa6.